docs/source/installation/for_dev.rst

```diff
@@ -22,7 +20,7 @@ Finally, after having cloned this repo, run the following:
 
    python3 -m pip install .
 
-Please see :doc:`contributing` for more details about contributing to Manim.
+Please see page :doc:`<../contributing>` for more details about contributing to Manim.
```

Comment:
```suggestion
Please see :doc:`our contributing guide<contributing>` for more details about contributing to Manim.
```

Reply:
I integrated your suggestion, although my change there was itself supposed to resolve a broken link. I'll investigate that further later on. Thank you!